src/views/RemoveLiquidity/index.tsx

```diff
@@ -99,7 +109,10 @@ export default function RemoveLiquidity() {
   async function onAttemptToApprove() {
     if (!pairContract || !pair || !library || !deadline) throw new Error('missing dependencies')
     const liquidityAmount = parsedAmounts[Field.LIQUIDITY]
-    if (!liquidityAmount) throw new Error('missing liquidity amount')
+    if (!liquidityAmount) {
+      toastError(t('Error'), t('Missing liquidity amount'))
```

Comment:
Hmm. It's good for now. But I think we could catch the error in global scope and call `toastError` in one place for all erros thrown